[DOCS] New page: Detection & Response dashboard (#2085)
elastic/security-docs: docs/getting-started/security-ui.asciidoc

* Host isolation exceptions
* Blocklist

Pages are grouped into these main sections within the navigation pane:

> * *Dashboards*: View and measure detections, investigations, and event trends across your environment.

Reviewer: ```suggestion
* *Dashboards*: Visualize detections, investigations, and event trends across your environment.
```

[float]
[[overview-ui]]
=== Overview dashboard

> The Overview dashboard provides a high-level snapshot view of detections, external alerts, and event trends. These trends are useful to assess overall system health and find anomalies that may require further investigation. Refer to <<overview-dashboard, Overview dashboard>> for more information.

Reviewer: ```suggestion
The Overview dashboard provides a high-level snapshot of detections, external alerts, and event trends. It can help you assess overall system health and find anomalies that may require further investigation. Refer to <<overview-dashboard, Overview dashboard>> for more information.
```

image::images/overview-pg.png[Overview dashboard]

[float]
[[detection-response-dashboard-ui]]
=== Detection & Response dashboard

> The Detection & Response dashboard provides focused visibility into the day-to-day operational activities of your security environment. Security operations managers and analysts can quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with frequent and severe alerts. Refer to <<detection-response-dashboard, Detection & Response dashboard>> for more information.

Reviewer: ```suggestion
The Detection & Response dashboard provides focused visibility into the day-to-day operations of your security environment. It helps security operations managers and analysts quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with alerts. Refer to <<detection-response-dashboard, Detection & Response dashboard>> for more information.
```
Just making the same suggestion I made earlier in the PR on another topic, to make it more consistent in case you decide to go with it.
